docs/ios/README.md

```diff
@@ -0,0 +1,31 @@
+# iOS distribution
+
+The source code can be found in [RNNativePlayground](https://github.com/kiwicom/mobile/tree/master/ios/RNNativePlayground)
+
+## Test playground app
+
+### KiwiHotels module
+
+1. To run `KiwiHotels` module with the packager, change [`RNKiwiConstants.m`](https://github.com/kiwicom/mobile/blob/master/ios/RNKiwiMobile/RNKiwiConstants.m) like follows:
+
+```objc
++ (NSURL *)bundleURL {
+  return [NSURL URLWithString:@"http://localhost:8081/app/native.bundle?platform=ios&dev=true"];
+}
+```
+
+2. Then run the packager from the root of this repo:
+
+```bash
+yarn start
+```
+
+3. Finally just open the project in XCode (use `reactNativeApp.xcworkspace`) and choose `RNNativePlayground` scheme.
```

Comment:
Nit: `Finally, open the project` (it's a good practice to avoid using "simply" and "just" as these things are never that easy!)

Nit: I would suggest changing "... and choose" to ". Make sure you have "RNNativePlayground" scheme selected before running. 

Would be great to link `scheme selected` to an Apple documentation where it shows how to do it with XCode. I assume there might be people using it that have little experience (is that wrong assumption?)